crates/solver/src/settlement_submission/submitter/custom_nodes_api.rs

```diff
@@ -64,8 +73,17 @@ impl TransactionSubmitting for CustomNodesApi {
                     });
                 }
                 Err(err) => {
-                    tracing::warn!(?err, ?lable, "single custom node tx failed");
-                    super::track_submission_success(lable.as_str(), false);
+                    // error is not real error if transaction pool responded that received transaction is already in the pool
+                    let real_error = match err.clone() {
```

Comment:
The error does not need to be cloned. This also works:
```rust
let real_error = match &err {
//...
```

Reply:
True, thanks.